distmv/general.go

```diff
@@ -0,0 +1,21 @@
+package distmv
+
+var (
+	ErrSizeMismatch  = "dist: size mismatch"
```

Comment:
There they are type error, these aren't. If you don't export, then you ca keep it this way.